index.js

```diff
@@ -205,7 +205,18 @@ Keystone.prototype.import = function(dirname) {
  */
 
 Keystone.prototype.applyUpdates = function(callback) {
-	require('./lib/updates').apply(callback);
+	var self = this;
+	self.callHook('pre:updates', function(err){
+		if(err){
+			callback(err);
+		}
+		require('./lib/updates').apply(function(err){
```

Comment:
`applyUpdates` and the callback to 'pre:updates' are both one-shots, so yeah it should be ok. Whether it's clean is debatable, but since it already had an inline `require` I left it as is.